src/app/portal/components/case/case-retention-date/case-retention-confirm/case-retention-confirm.component.html

```diff
@@ -14,12 +14,20 @@ <h1 class="govuk-heading-l">Check retention date change</h1>
           <td id="retain-date" class="govuk-table__cell table-width">
             {{ getDate() }}
           </td>
-          <td class="govuk-table__cell"><a href="#" id="date-link" (click)="onReturnDate($event)">Change</a></td>
+          <td class="govuk-table__cell">
+            <a href="#" id="date-link" class="govuk-link" (click)="onReturnDate($event)"
+              >Change <span class="govuk-visually-hidden">case retention date</span></a
```

Comment:
is 'case retention date' the correct description for this link?

Reply:
The screen reader will read 'Change case retention date'